### ISSUE TITLE
Abregado-Rae ground battle game breaking: Red

### ISSUE THREAD

**Issue by ArtemisFaraday** [soulrelic616/bfc1138.js#23]
While playing as the empire, I fought against the Mandalorians at Abregado-Rae. After I beat the Mandalorians the game crashed as it was loading up the battle results screen. I went back to my save and auto-resolved the battle and it also crashed at the battle results screen, so its either planet problems or more Mandalorian problems.

**Comment by Ardus-Kaine** [comment]
Issue exists in Republic at War, as well, it seems.

**Comment by soulrelic616** [comment]
Does it happen only with space and land battles? or is it an issue with both?

**Comment by ArtemisFaraday** [comment]
It happened in space battle as well, but that was because there was a
mandalorian battlecruiser in the fight as well, which meant I still could
auto resolve the space battle and be alright, but I can't find a way to
play a land battle on abregado rae without the game crashing
On Dec 23, 2015 07:48, "Bernardo Baldiviezo" <notifications@github.com>
wrote:
>
> Does it happen only with space and land battles? or is it an issue with
both?

**Comment by soulrelic616** [comment]
Right,  was the root of the problem the Mando Battlecruiser? If that's the case I think we might have a solution at hand.

**Comment by ArtemisFaraday** [comment]
For the space battles that was the problem, but a completely new problem
arrived when I played ground battle and the game crashed every time the
results screen was about to come up.

**Comment by Ardus-Kaine** [comment]
Did you try using different land units each time?

**Comment by ArtemisFaraday** [comment]
I wasn't able to as both of my save happened to be at the moment the battle
was taking place, so I just started a new game and let the ai take abregado
rae
